src/main/java/de/tum/in/www1/artemis/service/connectors/lti/LtiService.java

```diff
@@ -105,6 +109,15 @@ public void authenticateLtiUser(String email, String username, String firstName,
         // 2. Case: Lookup user with the LTI email address and make sure it's not in use
         if (artemisAuthenticationProvider.getUsernameForEmail(email).isPresent() || userRepository.findOneByEmailIgnoreCase(email).isPresent()) {
             log.info("User with email {} already exists. Email is already in use.", email);
+
+            if (trustExternalLTISystems) {
+                // If the email is already in use, but we trust external LTI systems, we authenticate the user with the email
```

Comment:
This comment is kind of unnecessary as we have a if and a log info before this line to describe the same things as the comment